drivers/storage/portworx/deployment.go

```diff
@@ -1295,9 +1295,39 @@ func (t *template) getVolumeMounts() []v1.VolumeMount {
 	for _, fn := range extensions {
 		volumeInfoList = append(volumeInfoList, fn()...)
 	}
+	volumeInfoList = t.overrideVolumeMount(volumeInfoList)
 	return t.mountsFromVolInfo(volumeInfoList)
 }
 
+func (t *template) overrideVolumeMount(volInfos []volumeInfo) []volumeInfo {
```

Comment:
Why is this called `overrideVolumeMount`?
* we are not _overriding_ the internal mounts-list
* instead, we are _filtering_ (and extending) the list that was provided as input

Reply:
we are not just filtering but also modifying the host path property inside the function